stock_quant_partner_info/migrations/8.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ok, it is a public module I've thought that it was required, just in case. Thanks for the explanation